locales/en-US/experiments.ftl

```diff
@@ -72,15 +72,17 @@ nomore404sContributors1Title = Developer, Wayback Machine, The Internet Archive
 nomore404sContributors2Title = Director, Wayback Machine, The Internet Archive
 nomore404sContributors3Title = Senior Crawl Engineer, The Internet Archive
 notesDescription = A simple notepad built into Firefox. Ever need to jot something down while you browse the web? This is the experiment for you.
-notesIntroduction = Notes adds a simple, one-page notepad to your browser’s sidebar. We’ve added a few basic formatting options to start, but we want your feedback to tell us what features you’d like to see us add.
+notesIntroductionMultinote = Notes adds a simple notepad to your browser’s sidebar. Sign into your Firefox Account to sync all of your notes with end-to-end encryption.
```

Comment:
Sign into ----> Sign in to
https://design.firefox.com/photon/copy/word-list.html#s

Reply:
Good catch, thanks!